shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -956,7 +962,7 @@ - (void)goToApplicationLifecycle:(nonnull NSString*)state {
     case UITouchTypeDirect:
     case UITouchTypeIndirect:
       return flutter::PointerData::DeviceKind::kTouch;
-    case UITouchTypeStylus:
+    case UITouchTypePencil:
```

Comment:
is this change intended?

Reply:
Yeah, UITouchTypeStylus is deprecated and is suggested https://developer.apple.com/documentation/uikit/uitouchtype/uitouchtypestylus